tools/bin/ecflow/processes/main/meteorological/combine.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+#####combine.py
+#####usage: <python> <combine.py> <configuration.cfg>
+
+###### This script runs tocel_combine.py
+
+import sys
+import pandas as pd
+import numpy as np
+import argparse
+import types
+import os
+import multiprocessing as mp
+from shutil import copyfile
+
+from tonic.io import read_config
+from monitor.io import replace, proc_subprocess
+from monitor.log import set_logger
+from monitor.share import LOG_LEVEL 
+
+######### ----------------------------------------###########
+
+
+def main():
+	# parse arguments
+	parser = argparse.ArgumentParser(description='Run VIC')
+	parser.add_argument('config_file', metavar='config_file',
+                        type=argparse.FileType('r'), nargs=1,
+                        help='configuration file')
+	args = parser.parse_args()
+	config_dict = read_config(args.config_file[0].name)
+
+	# set up logger
+	logger = set_logger(os.path.splitext(os.path.split(__file__)[-1])[0],
+                        LOG_LEVEL)
+
+	#read in variables from config file
+	cores = config_dict['ECFLOW']['Cores']
+	run_dir = config_dict['VIC']['RunDir']
+	log_dir_root = config_dict['VIC']['LogDirRoot']
+	forcing_names = config_dict['COMBINE']['Forcing_Name_List']
+	script = config_dict['COMBINE']['Script'] #location of original tocel_combine.py 
+	tmin_dir = config_dict['COMBINE']['Tmin_dir']
+	tmax_dir = config_dict['COMBINE']['Tmax_dir']
+	precip_dir = config_dict['COMBINE']['Precip_dir']
+	wind_dir = config_dict['COMBINE']['Wind_dir']
+	final_dir = config_dict['COMBINE']['Final_Met_dir']
+	
+	#set up multiprocessor
+	if cores > 1:
+        	pool = mp.Pool(processes=cores)
+        	logger.debug('started pool with {} processes'.format(cores))
+
+
+
+	#read in list of lat/lons
+	lat_lon = np.genfromtxt('%s' %(forcing_names), dtype='str')
```

Comment:
no reason to use string formatting here.  Just `np.genfromtxt(forcing_names, dtype='str') should work.